src/System.Private.Xml/src/System.Private.Xml.csproj

```diff
@@ -730,6 +719,11 @@
     <Compile Include="System\Xml\Xsl\Xslt\XslVisitor.cs" />
     <Compile Include="$(CommonPath)\System\LocalAppContext.cs" />
     <Compile Include="System\Xml\Core\LocalAppContextSwitches.cs" />
+    <Compile Include="System\CodeDom\CodeDomProvider.cs" />
+    <Compile Include="System\CodeDom\CodeGenerator.cs" />
+    <Compile Include="System\CodeDom\CompilerError.cs" />
+    <Compile Include="System\CodeDom\CompilerResults.cs" />
+    <Compile Include="System\CodeDom\TempFileCollection.cs" />
```

Comment:
These are few types that lived in System.CodeDom which if I remove, some functionalities of Xml would be broken. Since these could be separated from the rest of CodeDom and add value to Xml, I kept them as internal classes.